linux/net/rina/du-protection.h

```diff
@@ -32,4 +33,16 @@ bool    dup_ttl_set(struct pdu_ser * pdu, size_t value);
 ssize_t dup_ttl_decrement(struct pdu_ser * pdu);
 bool    dup_ttl_is_expired(struct pdu_ser * pdu);
 
+int dup_encrypt_data(const char              * src,
```

Comment:
Shouldn't this follow the same convention as the function signatures above? E.g. use pdu_ser instead of char *.